tsconfig.json

```diff
@@ -1,17 +1,16 @@
 {
     "compilerOptions": {
         /* Basic Options */
-        "target": "esnext" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
-        "noEmit": true /* Do not emit outputs. We use babel as our compiler. */,
-        "isolatedModules": true /* Transpile each file as a separate module (similar to 'ts.transpileModule'). This is how babel operates */,
-        "lib": [
-            /* Specify library files to be included in the compilation. */
-            "dom" /* We have a browser environment */
+        "target": "esnext"                        /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
+        "noEmit": true                            /* Do not emit outputs. We use babel as our compiler. */,
+        "isolatedModules": false                  /* Transpile each file as a separate module (similar to 'ts.transpileModule'). This is how babel operates */,
```

Comment:
I guess the comment no longer applies to the setting, right?

And this now opens up the possibility of the typecheck passing, and the compiling with babel to fail. If I understood correctly. Thats not great, but, might not be a huge issue as we build the project on the CI server, so it will not actually go into master.

Reply:
You're right. I'll change it.